<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>662</width>
    <height>793</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Query Tool</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_7">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="0" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Headers</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="1" colspan="3">
           <widget class="QListView" name="headerListView">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>55</height>
             </size>
            </property>
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <property name="showDropIndicator" stdset="0">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="1" colspan="3">
           <widget class="QPushButton" name="addHeaderButton">
            <property name="text">
             <string>Manage Headers</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Method</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1" colspan="3">
           <widget class="QComboBox" name="methodComboBox">
            <item>
             <property name="text">
              <string>GET</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>POST</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>PUT</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>DELETE</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>HEAD</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>OPTIONS</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>PATCH</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Address</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QComboBox" name="schemeComboBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <item>
             <property name="text">
              <string>HTTP</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>HTTPS</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QLabel" name="label_6">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>://</string>
            </property>
           </widget>
          </item>
          <item row="3" column="3">
           <widget class="QLineEdit" name="addressInput"/>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Body</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="4" column="1" colspan="3">
           <widget class="QTextEdit" name="queryInput">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Return</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="5" column="1" colspan="3">
           <widget class="QTabWidget" name="resultsTabWidget">
            <property name="currentIndex">
             <number>1</number>
            </property>
            <widget class="QWidget" name="treeTab">
             <attribute name="title">
              <string>Tree View</string>
             </attribute>
             <layout class="QGridLayout" name="gridLayout_4">
              <item row="0" column="0">
               <widget class="QTreeView" name="jsonTreeView"/>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="textTab">
             <attribute name="title">
              <string>Plain Text</string>
             </attribute>
             <layout class="QGridLayout" name="gridLayout_5">
              <item row="0" column="0">
               <widget class="QTextEdit" name="resultTextEdit"/>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="responseTime">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="6" column="1" colspan="3">
           <widget class="QPushButton" name="queryButton">
            <property name="text">
             <string>Query</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>662</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSave_Query">
   <property name="text">
    <string>&amp;Save Query</string>
   </property>
  </action>
  <action name="actionLoad_Query">
   <property name="text">
    <string>&amp;Load Query</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>&amp;Settings</string>
   </property>
  </action>
  <action name="actionUpload_Route">
   <property name="text">
    <string>&amp;Upload Route</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
